lib/extty.ex

```diff
@@ -114,11 +114,28 @@ defmodule ExTTY do
     {Elixir.IEx, :start, opts}
   end
 
+  defp shell_spawner(%{type: :muontrap, shell_opts: opts}) do
```

Comment:
```suggestion
  defp shell_spawner(%{type: :system, shell_opts: opts}) do
```
MuonTrap is an implementation detail. What if we generalize this to System?

Reply:
How about letting the user pass a `:custom` shell spawner that calls an arbitrary function?

I'd be for adding an example to the docs if that's direction that you go since I wouldn't have expected calling `System.cmd` to work well with this. I'm thrilled that it does, though.